public/app/features/dashboard/state/PanelModel.ts

```diff
@@ -28,6 +28,7 @@ import { CoreEvents } from '../../../types';
 
 export const panelAdded = eventFactory<PanelModel | undefined>('panel-added');
 export const panelRemoved = eventFactory<PanelModel | undefined>('panel-removed');
+export const snapshotCreated = eventFactory('snapshot-created');
```

Comment:
This needs to follow the pattern above
```suggestion
export const snapshotCreated = eventFactory<PanelModel | undefined>('snapshot-created');
```
```suggestion
export const snapshotCreated = eventFactory('snapshot-created');
```

Reply:
Done 👍